src/createRollupConfig.ts

```diff
@@ -154,6 +154,8 @@ export async function createRollupConfig(
             '**/*.test.ts',
             '**/*.spec.tsx',
             '**/*.test.tsx',
+            // storybook files
+            '**/*.stories.tsx',
```

Comment:
I wonder if we need to do `...(!!opts.tsconfig.exclude ? [opts.tsconfig.exclude] : [])`

Reply:
I tested the [logic](https://github.com/ezolenko/rollup-plugin-typescript2/blob/master/src/parse-tsconfig.ts#L44) in rpts2:
<img width="647" alt="Screen Shot 2020-04-07 at 1 53 08 AM" src="https://user-images.githubusercontent.com/9523719/78649675-9f279e00-7872-11ea-807b-70ea715da27a.png">

I don't see an issue... it should work fine.

Although strangely enough a different merge loses some options which I'm surprised get lost...
<img width="628" alt="Screen Shot 2020-04-07 at 1 56 39 AM" src="https://user-images.githubusercontent.com/9523719/78649994-24ab4e00-7873-11ea-82e9-93184de0ec89.png">

Reply:
There exists a few lines after L44 in that rpts2 file I linked which I believe also do some merging.

Reply:
That's not because of rpts2 -- that's how `_.merge` works with arrays. `exclude` is an array, so your custom exclude would just override the same indices.
In your first "test", `**/*.stories.tsx` also overrides the first entry, `**/*.spec.ts`

I think this jogged my brain a bit -- I'm pretty sure that line in rpts2 is around where the bug is.
TSDX has similar logic as rpts2 using TS's custom parsers to read a `tsconfig` and then get its `compilerOptions` through all `extends` chains. But that's only for `compilerOptions`, meaning `exclude` etc from an `extends` aren't actually parsed.

Problem there is that I don't know if there's a way to get the rest. This part of the TS API isn't documented but is used in lots of places in the wild for parsing, it took me a while to figure out how to do it myself, c.f. #489

Reply:
Ah, the original config gets mutated with the new `include`/`exclude`, while `compilerOptions` is unchanged and returned per https://github.com/facebook/create-react-app/pull/5537

I'll have to look into rpts2 config and see if there's actually a bug or not, I'm not sure. I still haven't repro'd #663